docs/manual/_renderer.py

```diff
@@ -0,0 +1,19 @@
+from typing import Optional, Union
+
+from griffe import dataclasses as dc
+from plum import dispatch
+from quartodoc import MdRenderer
+
+
+class Renderer(MdRenderer):
+    style = 'mystyle'
+
+    @dispatch
+    def signature(
+        self,
+        el: Union[dc.Class, dc.Function],
+        source: Optional[dc.Alias] = None
```

Comment:
Is there a reason for adding this parameter?

Reply:
it's an overloaded function, so the caller (which is quartodoc) may call the function with that last argument.

Reply:
Oops, my bad, I missed that parameter in their docs.